README.md

```diff
@@ -64,5 +77,4 @@ Give a ⭐️ if you like this project!
 
 ## 📝 License
 
-This project is [MIT](./MIT.md) licensed.
-
+Copyright 2021, [Addis Tsega]
```

Comment:
### [Optional]

- [ ] It would be nice if you put **This project is [MIT](./MIT.md) licensed.** in place of **Copyright 2021, [Addis Tsega]** :relaxed:

Also, you need to create a MIT.md file in your main directory :blush:. Please follow the steps to create a MIT.md file. It is your license file for this project so please take it seriously :thinking:
- Create a MIT.md file in your main directory
- Put the contents from this [template](https://github.com/microverseinc/readme-template/blob/master/MIT.md) into your MIT.md file
- Place your name in place of **[YOUR NAME]** and put the current year and also in **[APP TYPE]** you need to add **WEBSITE** or **WEB APP** :+1: